crates/testing/tests/tests_1/upgrade_task.rs

```diff
@@ -255,7 +255,7 @@ async fn test_upgrade_and_consensus_task() {
         upgrade_vote_recvs,
         vec![QuorumProposalRecv(proposals[1].clone(), leaders[1])],
         vec![
-            DACertificateRecv(dacs[1].clone()),
+            VIDShareRecv(get_vid_share(&vids[1].0, handle.get_public_key())),
```

Comment:
Not sure if this is intentional. Why do we replace `DACertificateRecv` with `VIDShareRecv` rather than just add `VIDShareRecv`?

Reply:
Oh this is because we don't need the DAC to propose, but we now do need the VID share to propose.